app.rb

```diff
@@ -0,0 +1,101 @@
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def display_books
+    @books.each do |book|
+      puts "Title: '#{book.title}', Author: #{book.author}"
+    end
+  end
+
+  def display_people
+    @people.each do |person|
+      puts "Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
```

Comment:
- [ ] Kindly make sure the that when you display people add the kind of person he/she is(student or teacher) just like in the video example.

How you display | the video template
---|----
<img width="377" alt="image" src="https://github.com/matee-safi/School-library/assets/65068771/ee11c139-2eee-49d3-8323-1d8a1e637add"> | <img width="377" alt="image" src="https://github.com/matee-safi/School-library/assets/65068771/b6b1035e-b2a4-4c44-a135-267324602902">